libs/core/tests/integration/containers/set_intersection.cpp

```diff
@@ -47,10 +47,10 @@ TEST(SetIntersection, BasicSetMapCheck1)
   auto const c = a & b;
 
   EXPECT_EQ(c.size(), 4);
-  EXPECT_TRUE(core::IsIn(c, 3));
-  EXPECT_TRUE(core::IsIn(c, 4));
-  EXPECT_TRUE(core::IsIn(c, 5));
-  EXPECT_TRUE(core::IsIn(c, 6));
+  EXPECT_TRUE(meta::IsIn(c, 3));
```

Comment:
Ideally the meta library was supposed to just be a compile time meta programming helpers. How come this was necessary to move it?